framework_tvb/tvb/interfaces/web/templates/jinja2/base_template.html

```diff
@@ -49,6 +49,8 @@
         <script type="text/javascript" src="{{ deploy_context | safe }}/static/jquery/jquery.json.js"></script>
         <script type="text/javascript" src="{{ deploy_context | safe }}/static/js/genericTVB.js?{{ currentVersionJS }}"></script>
 
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+
```

Comment:
Please don't use external dependencies of this sort. Tvb web site needs to function also in a local context with poor or no connectivity

Reply:
I have removed external dependencies in the latest commit and added the icons locally.